.github/workflows/lint-test.yaml

```diff
@@ -9,15 +9,17 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
+      - name: Run chart-testing (lint)
+        id: lint
+        uses: scottrigby/chart-testing-action@help-kind-run-only-if-changed
```

Comment:
seems to do the trick (https://github.com/helm/charts-repo-actions-demo/commit/ba9a59f761584bd09602be584e56f8ad9ca14cb7/checks?check_suite_id=388497336) - so maybe just update this PR with the official helm action and then we merge the two successively?

Reply:
OK merged and created next pre-release tag https://github.com/helm/chart-testing-action/releases/tag/v1.0.0-alpha.3 and updated to official helm action release in 4dcf446 .